From dfe965eb139737af4c095fbadbfb7a4f901b9439 Mon Sep 17 00:00:00 2001
From: "clopez@igalia.com"
 <clopez@igalia.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Mon, 19 Mar 2018 22:05:17 +0000
Subject: [PATCH xserver] REGRESSION(r225340): WPE port should not be getting
 EGL X11 types on ANGLE https://bugs.webkit.org/show_bug.cgi?id=183749

Reviewed by Alex Christensen.

Add back the USE_WPE ifdef.

* changes.diff:
* include/EGL/eglplatform.h:

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@229721 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 Source/ThirdParty/ANGLE/include/EGL/eglplatform.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/ThirdParty/ANGLE/include/EGL/eglplatform.h b/Source/ThirdParty/ANGLE/include/EGL/eglplatform.h
index 333448be348..47d18093c96 100644
--- a/Source/ThirdParty/ANGLE/include/EGL/eglplatform.h
+++ b/Source/ThirdParty/ANGLE/include/EGL/eglplatform.h
@@ -100,7 +100,7 @@ typedef struct ANativeWindow*           EGLNativeWindowType;
 typedef struct egl_native_pixmap_t*     EGLNativePixmapType;
 typedef void*                           EGLNativeDisplayType;
 
-#elif defined(USE_OZONE)
+#elif defined(USE_OZONE) || defined(USE_WPE)
 
 typedef intptr_t EGLNativeDisplayType;
 typedef intptr_t EGLNativeWindowType;
-- 
2.11.0

